ansible/group_vars/all.yml

```diff
@@ -40,6 +40,8 @@ insecure_registrys:
 # The port that the Kubernetes apiserver component listens on.
 kube_master_api_port: 443
 
+# TODO: Add kube_apiserver_vip for multi-master configurations
```

Comment:
Replace

# TODO: Add kube_apiserver_vip for multi-master configurations

with

# The virtual IP address used in multi-master configurations. kube_apiserver_vip
# should be an available IP address on the Kubernetes network.